scripts/update-index-alias.py

```diff
@@ -0,0 +1,107 @@
+#!/usr/bin/env python
+"""
+A script to update the search index an alias is assoiciated with.
+
+To delete old indexes (those with no alias), set --delete-old-indexes=yes
+
+Usage:
+scripts/update-index-alias.py <alias> <target> <stage> <search-api-endpoint> [options]
+
+Options:
+    --delete-old-indexes=<delete-old-indexes>  [default: ]
```

Comment:
Oh, I see, that's a specific piece of syntax for docopt. Meh. Have you used argparse? I generally prefer it for writing scripts, but it does preclude having the syntax documentation at the top of the script.